service/frontend/workflow_handler.go

```diff
@@ -3070,7 +3071,12 @@ func (wh *WorkflowHandler) validateStartWorkflowArgsForSchedule(
 		return errIDReusePolicyNotAllowed
 	}
 
-	if err := wh.validateSearchAttributes(startWorkflow.GetSearchAttributes(), namespaceName); err != nil {
+	// Unalias startWorkflow search attributes only for validation. Keep aliases in request.
```

Comment:
```suggestion
	// Unalias startWorkflow search attributes only for validation. Keep aliases in the request, because the request will be sent back to frontend to start workflows, which will unalias at that point.
```